docs/migrate.ipynb

```diff
@@ -254,7 +254,45 @@
       "source": [
         "## Converting your code\n",
         "\n",
-        "Converting existing `tf.summary` usage to the TF 2.x API cannot be reliably automated, so the [`tf_upgrade_v2` script](https://www.tensorflow.org/guide/upgrade) just rewrites it all to `tf.compat.v1.summary`.  To migrate to TF 2.x, you'll need to adapt your code as follows:"
+        "Converting existing `tf.summary` usage to the TF 2.x API cannot be reliably automated, so the [`tf_upgrade_v2` script](https://www.tensorflow.org/guide/upgrade) just rewrites it all to `tf.compat.v1.summary` and will not enable the TF 2.x behaviors automatically."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "1972f8ff0073"
+      },
+      "source": [
+        "### Partial Migration\n",
+        "\n",
+        "It is possible to invoke TF 2.x behaviors within <a href=\"https://www.tensorflow.org/api_docs/python/tf/compat/v1/summary\"><code>tf.compat.v1.summary</code></a> APIs under the following conditions:\n",
+        "\n",
+        " - The outermost context is eager mode\n",
+        " - A suitable TF2 summary writer exists\n",
+        " - A non-empty value for step is set for the writer (using <a href=\"https://www.tensorflow.org/api_docs/python/tf/summary/SummaryWriter#as_default\"><code>tf.summary.SummaryWriter.as_default</code></a>, or alternatively <a href=\"https://www.tensorflow.org/api_docs/python/tf/summary/experimental/set_step\"><code>tf.summary.experimental.set_step</code></a>)\n",
+        "\n",
+        "Note that when TF2 summary implementation is invoked, the return value will be an empty bytestring tensor, to avoid duplicate summary writing. Additionally, the input argument forwarding is best-effort and not all arguments will be preserved (`family` argument will be supported whereas `collections` will be removed).\n",
```

Comment:
I might add "for example" at the start of the parenthetical, just to be clear it's an example and that the family argument isn't the only one being preserved.

Reply:
Done.